Assets/MixedRealityToolkit/Utilities/MixedRealityServiceRegistry.cs

```diff
@@ -23,6 +25,31 @@ public static class MixedRealityServiceRegistry
         private static Dictionary<Type, List<KeyValuePair<IMixedRealityService, IMixedRealityServiceRegistrar>>> registry =
             new Dictionary<Type, List<KeyValuePair<IMixedRealityService, IMixedRealityServiceRegistrar>>>();
 
+        /// <summary>
+        /// A cache used to power <seealso cref="GetAllServices(IMixedRealityServiceRegistrar)"/>
```

Comment:
Nit: power the registrar?

Reply:
Well kinda... the comment as-is is more accurate than just "power the registrar." It powers that specific function, not the entire registrar.